packages/edit-site/src/components/global-styles/screen-link-color.js

```diff
@@ -29,40 +34,87 @@ function ScreenLinkColor( { name } ) {
 		isLinkEnabled &&
 		( solids.length > 0 || areCustomSolidsEnabled );
 
-	const [ linkColor, setLinkColor ] = useStyle(
-		'elements.link.color.text',
-		name
-	);
-	const [ userLinkColor ] = useStyle(
-		'elements.link.color.text',
-		name,
-		'user'
-	);
+	const pseudoStates = {
+		default: {
+			...useStyle( 'elements.link.color.text', name ).reduce(
+				( acc, psuedoSelector, index ) => {
+					acc[ index === 0 ? 'value' : 'handler' ] = psuedoSelector;
+					return acc;
+				},
+				{}
+			),
+			userValue: useStyle(
+				'elements.link.color.text',
+				name,
+				'user'
+			)[ 0 ],
+		},
+		hover: {
+			...useStyle( 'elements.link.:hover.color.text', name ).reduce(
+				( acc, psuedoSelector, index ) => {
+					acc[ index === 0 ? 'value' : 'handler' ] = psuedoSelector;
+					return acc;
+				},
+				{}
+			),
+			userValue: useStyle(
+				'elements.link.:hover.color.text',
+				name,
+				'user'
+			)[ 0 ],
+		},
+	};
 
 	if ( ! hasLinkColor ) {
 		return null;
 	}
 
+	const tabs = Object.keys( pseudoStates ).map( ( pseudoSelector ) => {
+		return {
+			name: pseudoSelector,
+			title: ucFirst( pseudoSelector ),
```

Comment:
We are generating this dynamically but I assume it will still need to be translatable. I'll need to revisit the docs on how to do this.